protoc-gen-swift.hcl

```diff
@@ -0,0 +1,15 @@
+description = "Plugin for using protobuf with Swift"
+binaries = ["protoc-gen-swift"]
+test = "protoc-gen-swift --version"
+
+darwin {
+  source = "https://github.com/cashapp/hermit-build/releases/download/protoc-gen-swift/protoc-gen-swift-${version}-macos.tar.gz"
```

Comment:
Have to re-run the job (or rename the existing release) and change this to use the right release tag?